tests/test_git_repo_scanner.py

```diff
@@ -61,24 +61,26 @@ def test_load_repo_does_not_filter_submodules_when_requested(
     def test_extra_inclusions_get_added(self, mock_load: mock.MagicMock):
         mock_load.return_value = (
             self.data_dir / "pyproject.toml",
-            {"include_paths": "include-files", "include_path_patterns": ("foo/",)},
+            {"include_path_patterns": ("foo/",)},
         )
+        self.global_options.include_path_patterns = ("foo/",)
         test_scanner = scanner.GitRepoScanner(
             self.global_options, self.git_options, str(self.data_dir)
         )
         test_scanner.load_repo("../tartufo")
         self.assertCountEqual(
             test_scanner.included_paths,
-            [re.compile("foo/"), re.compile("tartufo/"), re.compile("scripts/")],
+            [re.compile("foo/")],
```

Comment:
This test doesn't seem to me like it is proving the same intent anymore. Especially now that you added the `self.global_options.include_path_patterns = ("foo/",)`

Now, this test only shows that we load patterns from global options. The intent of this test is to show that:

* `global_options` include `"foo/"`
* `load_repo` is called with a separate repository, with a config file in it.
* That config file includes `"bar/"`
* The `included_paths` must now contain both `"foo/"`, and `"bar/"`